libcloud/common/base.py

```diff
@@ -486,6 +486,7 @@ def user_agent_append(self, token):
 
     def request(self, action, params=None, data=None, headers=None,
                 method='GET', raw=False, stream=False):
+
```

Comment:
looks like you added a blank line before the docstring, travis will probably fail this